lib/csv_to_popolo/version.rb

```diff
@@ -1,3 +1,3 @@
 module Popolo_CSV
-  VERSION = "0.9.2"
+  VERSION = '0.9.3'
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.